Dockerfile

```
@@ -28,9 +28,16 @@ USER rsk
 WORKDIR /var/lib/rsk
 COPY --from=build --chown=rsk:rsk /home/rsk/rsk.jar ./
 
-ENTRYPOINT ["java", \
-    "-Drpc.providers.web.http.bind_address=0.0.0.0", \
-    "-Drpc.providers.web.http.hosts.0=localhost", \
-    "-Drpc.providers.web.http.hosts.1=127.0.0.1", \
-    "-Drpc.providers.web.http.hosts.2=::1", \
-    "-cp", "rsk.jar", "co.rsk.Start"]
+ENV JAVA_OPTS=""
+ENV RSKJ_CLASS=co.rsk.Start
+ENV RSKJ_OPTS=""
+
+ENTRYPOINT java \
+    $JAVA_OPTS \
+    -Drpc.providers.web.http.bind_address=0.0.0.0 \
+    -Drpc.providers.web.http.hosts.0=localhost \
+    -Drpc.providers.web.http.hosts.1=127.0.0.1 \
+    -Drpc.providers.web.http.hosts.2=::1 \
+    -cp rsk.jar \
+    $RSKJ_CLASS \
+    $RSKJ_OPTS
```

I'd rather use `DEFAULT_JVM_OPTS` name like [here](https://github.com/rsksmart/rskj/blob/6ebaacfaa486b5086b5f2a3e70adb4cb0057675f/gradlew#L89), as `JAVA_OPTS` seems to be a semi-standard jvm env. variable name and may collide in some scenarious.

Also I believe that the default amount of ram being provided to a java process may not be enough for an RSKj node to operate - so what about defining it something like this?

```suggestion
ENV DEFAULT_JVM_OPTS='"-Xms3G" "-Xmx5G"'
ENV RSKJ_SYS_PROPS='"-Drpc.providers.web.http.bind_address=0.0.0.0" "-Drpc.providers.web.http.hosts.0=localhost" "-Drpc.providers.web.http.hosts.1=127.0.0.1" "-Drpc.providers.web.http.hosts.2=::1"'
ENV RSKJ_CLASS=co.rsk.Start
ENV RSKJ_OPTS=""

ENTRYPOINT java \
    $DEFAULT_JVM_OPTS \
    $RSKJ_SYS_PROPS \
    -cp rsk.jar \
    $RSKJ_CLASS \
    $RSKJ_OPTS
```

note: I haven't tried that - it's just pseudocode, so needs to be tested

---

Addressed